converters/fromZigbee.js

```diff
@@ -4726,6 +4726,43 @@ const converters = {
             });
         },
     },
+    tuya_data_point_dump_alt: {
+        cluster: 'manuSpecificTuya',
+        type: ['commandActiveStatusReportAlt'],
+        convert: (model, msg, publis, options, meta) => {
+            // Don't use! This is only for tests with Fantem ZB006-X
+            // tuya_data_point_dump_alt will be removed in future!
+            // Changes will be integrated in tuya_data_point_dump, if tests successful
+            const getHex = (value) => {
+                let hex = value.toString(16);
+                if (hex.length < 2) {
+                    hex = '0' + hex;
+                }
+                return hex;
+            };
+            const now = Date.now().toString();
+            let dataStr = '';
+            for (const [i, dpValue] of msg.data.dpValues.entries()) {
+                tuya.logDataPoint('tuya_data_point_dump_alt', msg, dpValue, meta);
+                dataStr +=
+                    now + ' ' +
+                    meta.device.ieeeAddr + ' ' +
+                    getHex(msg.data.seq) + ' ' +
+                    getHex(i) + ' ' +
+                    getHex(dpValue.dp) + ' ' +
+                    getHex(dpValue.datatype);
+
+                dpValue.data.forEach((elem) => {
+                    dataStr += ' ' + getHex(elem);
+                });
+                dataStr += '\n';
+            }
+            const fs = require('fs');
+            fs.appendFile('data/tuya.dump.alt.txt', dataStr, (err) => {
```

Comment:
Please remove this, can cause issues on systems which are not writeable

Reply:
Tuya dump write to file is for debugging only, thus not meant for production systems and debugging is usually not done on read-only systems.
Same file writes are in function `tuya_data_point_dump` right above in code. Also logs to file and is for debugging only.
Would be very helpful for further debugging if this could stay in code.
Pls. consider merging.

Reply:
But if I merge this it will be used in production, so therefore I cannot merge it in this state.

Reply:
Removed write to fs